libs/permissions/permission-manager-impl/src/main/kotlin/net/corda/libs/permissions/manager/impl/PermissionUserManagerImpl.kt

```diff
@@ -37,6 +37,7 @@ import net.corda.schema.configuration.ConfigKeys
 import java.time.Instant
 import java.time.temporal.ChronoUnit
 import java.util.concurrent.atomic.AtomicReference
+import javax.persistence.EntityNotFoundException
```

Comment:
```suggestion
import net.corda.libs.permissions.common.exception.EntityNotFoundException
```
we might need to add a dependency on a module like so:
`implementation project(':libs:permissions:permission-common')`